Source/DiabloUI/diabloui.cpp

```diff
@@ -58,6 +59,44 @@ OptionalOwnedClxSpriteList ArtCursor;
 bool textInputActive = true;
 std::size_t SelectedItem = 0;
 
+const string_view BannedNames[] = {
+	"gvdl",
+	"dvou",
+	"tiju",
+	"cjudi",
+	"bttipmf",
+	"ojhhfs",
+	"cmj{{bse",
+	"benjo",
+	"cfbofs",
+	"dijol",
+	"dppo",
+	"ejlf",

```

Comment:
Do we care about false positives, like "Raccoon" or "Klondike"? Or is it better to be aggressive and simply have the player come up with a different name anyway?

Reply:
Probably best to avoid false positives.